install_config/install/prerequisites.adoc

```diff
@@ -249,6 +252,12 @@ connections when using standalone etcd (clustered).
 | *4789*
 |Required for SDN communication between pods on separate hosts.
 
+| *5404* or *5405*
+|Used with pacemaker HA.
```

Comment:
We should highlight that these ports are associated with Corosync and that they are UDP ports. Also, pacemaker can be highlighted like his pal native \`\*pacemaker\*\`.